corehq/apps/style/management/commands/fix_less_imports_collectstatic.py

```diff
@@ -21,24 +21,29 @@ def handle(self, *args, **options):
             with codecs.open(less_file, 'r', 'utf-8') as fd:
                 content = fd.read()
                 if content is not None:
-                    B3_REGEX = r"@\{b3-import-[a-z]+-[a-z]+\}"
-                    p = re.search(B3_REGEX, content)
-                    if p is not None:
-                        path_def = p.group(0)
-                        path_file = path_def.split('-')[-2]
-                        new_path = settings.LESS_B3_PATHS.get(path_file)
-                        if new_path is not None:
-                            print ("[CORRECTING] %(f_name)s path in %(less_file)s: "
-                                   "%(new_path)s" % {
-                                       'f_name': path_file,
-                                       'less_file': less_file,
-                                       'new_path': new_path,
-                                   })
-                            content = re.sub(B3_REGEX, new_path, content)
+                    BOWER_PATH =  '../../../../../../bower_components'
```

Comment:
haha. can we use an absolute path here? `os.path.join(settings.BASE_DIR, bower_components)`